### ISSUE TITLE
[Feature Request] Hide "not syncing" banner or make it subtler 

### ISSUE THREAD

**Issue by data-flux** [mb1986/rm-hacks#182]
For those of us without connect, the "Not Syncing" banner on almost every document is a bit intrusive.

The option to hide these banners or minimize them to a small icon would be most welcome.

**Comment by mb1986** [comment]
Could you please give me some more input? Is it exactly "Not Syncing"? I cannot find any string like it.

**Comment by data-flux** [comment]
Yes, like in this [reddit post](https://i.redd.it/igvyg3a7ybc91.jpg)

**Comment by mb1986** [comment]
Ok, I've found it, thank you.

**Comment by sidkoth1996** [comment]
Hi, just want to check if this is added to version 0.10 or set for a later release?
